### ISSUE TITLE
Add .nvmrc into gulp generator

### ISSUE THREAD

**Issue by mkornatz** [onedesign/generator-one-base#151]
nvm doesn't use the `engines` definition in `package.json`

**Comment by cmalven** [comment]
Good idea. The trick is to avoid hard-coding the value in this .npmrc and having to change it over time. Ideal might be that it automatically uses the current LTS release at the time the generator is run. Or we just make a point to update this every 6 months.

**Comment by mkornatz** [comment]
Maybe we can prompt for this when installing?

**Comment by MikeMcChillin** [comment]
Love the idea of getting a prompt of like 3 commonly used node versions, as well as maybe the option to add my own version.

**Comment by cmalven** [comment]
There's always this: https://www.npmjs.com/package/nodejs-latest
